flow/Trace.h

```diff
@@ -45,14 +45,15 @@ inline static bool TRACE_SAMPLE() { return false; }
 extern thread_local int g_trace_depth;
 
 enum Severity {
-	SevSample=1,
-	SevDebug=5,
-	SevInfo=10,
-	SevWarn=20,
-	SevWarnAlways=30,
-	SevError=40,
-	SevMaxUsed=SevError,
-	SevMax=1000000
+	SevNoInfo = 0,
```

Comment:
I would recommend naming this something like `SevVerbose`, as it's still legal for someone to enable these messages. If we instead wanted to outright disallow them (with additional changes required), we could name it `SevDisabled`, `SevHidden`, etc.